ERROR: Unable to locate a modulefile for 'python3/3.8.11'
ERROR: Unable to locate a modulefile for 'cuda/8.0'
Loading cudnn/v7.0-prod-cuda8
  Module ERROR: couldn't read file "/apps/dcc/etc/ModulesSL64/modulebase/cudnn":
    no such file or directory
        while executing
    "source /apps/dcc/etc/ModulesSL64/modulebase/cudnn"
        (file "/apps/dcc/etc/ModulesAlma92/modulefiles/cudnn/v7.0-prod-cuda8"
    line 9)
    Please contact <root@localhost>
ERROR: Unable to locate a modulefile for 'ffmpeg/4.2.2'
Loaded module: cuda/11.8
Traceback (most recent call last):
  File "/zhome/3e/b/164967/Documents/HPC_02613/malaka_performance/source_PFCA/./Try_v2.py", line 73, in <module>
    all_u = np.concatenate([r.get() for r in results_list])
TypeError: 'ApplyResult' object is not iterable

real	0m1.601s
user	0m0.352s
sys	0m0.425s
